1.9/administration/installing/custom/configuration-parameters.md

```diff
@@ -234,6 +234,17 @@ This parameter specifies whether to enable sharing of anonymous data for your cl
 
 If you’ve already installed your cluster and would like to disable this in-place, you can go through an [upgrade][3] with the same parameter set.
 
+### cosmos_config
+This parameter specifies a dictionary of packaging configuration to pass to Cosmos, DC/OS package
```

Comment:
...`packaging configuration to pass to the [DC/OS package manager](https://github.com/dcos/cosmos).